### DISCUSSION TITLE
Resilience4j is a fault tolerance library designed for Java8 and functional programming

### DISCUSSION THREAD

**Discussion by nagkumar** [resilience4j/resilience4j#2004] [General]
`Resilience4j is a fault tolerance library designed for Java8 and functional programming
`
Is this only for java8 or is it for any jdk above version 8 i.e. inclusive jdk 19, 20 etc..

**Comment by nagkumar** [comment]
![image](https://github.com/resilience4j/resilience4j/assets/332234/34bde67a-6fe6-4460-aec0-c8d4a02c63ec)
I am referring to this text

**Comment by nagkumar** [comment]
while intro content says jdk17 and above content says jdk8

![image](https://github.com/resilience4j/resilience4j/assets/332234/96eb21ce-378b-49a8-8fbb-bd79649fa3e4)

**Comment by RobWin** [comment]
Yes I need to fix it.
Resilience4j 1.x requires at least JDK8
Resilience4j 2.x requires at least JDK17
